fr/lecons/analyse-reseau-python.md

```diff
@@ -0,0 +1,574 @@
+Alexander Skene of Newtyle,local politician and author,male,1621,1694,10011149
+James Logan,colonial official and scholar,male,1674,1751,10007567
+Dorcas Erbery,Quaker preacher,female,1656,1659,10003983
+Lilias Skene,Quaker preacher and poet,male,1626,1697,10011152
+```
+
+Dans un fichier au format `.csv`, la virgule est le séparateur par défaut permettant de créer des colonnes lorsqu'on importe le fichier dans un tableur ou un logiciel d'analyse de données.
+
+Lorsque vous ouvrez le fichier des liens, vous constatez que nous utilisons les noms présents dans le fichier des sommets pour identifier ces derniers. Les liens commencent à un sommet &laquo;&nbsp;source&nbsp;&raquo; (`Source`) et se terminent à un sommet &laquo;&nbsp;cible&nbsp;&raquo; (`Target`). Bien que ces termes soient issus de l'analyse des réseaux dits &laquo;&nbsp;orientés&nbsp;&raquo;, nous considérons nos données comme un réseau &laquo;&nbsp;non orienté&nbsp;&raquo;&nbsp;: si une personne A connait une personne B, alors cette dernière doit également connaitre la personne A. Dans les réseaux orientés, les relations ne sont pas nécessairement réciproques (une personne A peut envoyer une lettre à B sans en recevoir une en retour)&nbsp;; dans les réseaux non orientés, les liens sont toujours réciproques ou &laquo;&nbsp;symétriques&nbsp;&raquo; (le lien peut également être porteur d'une intensité ou d'un poids, généralement appelé &laquo;&nbsp;weight&nbsp;&raquo; en anglais, on parle alors de réseau valué ou pondéré). Étant donné qu'il s'agit d'un réseau de connaissances, considérer que les relations sont non orientées est le plus approprié. Les relations symétriques dans un réseau non orienté sont utiles chaque fois qu'il s'agit de relations qui jouent le même rôle pour les deux parties. Deux ami·es ont une relation symétrique&nbsp;: iels sont chacun l'ami·e de l'autre. L'auteur ou l'autrice et le ou la destinataire d'une lettre ont une relation asymétrique car chacun·e joue un rôle différent. Les réseaux orientés et non orientés offrent chacun leurs potentialités (et parfois leurs mesures spécifiques) et vous devez choisir celui qui convient le mieux au type de relations que vous collectez et aux questions auxquelles vous voulez répondre. Voici les premiers liens du réseau non orienté des quakers&nbsp;:
+
+```
+Source,Target
+George Keith,Robert Barclay
+George Keith,Benjamin Furly
+George Keith,Anne Conway Viscountess Conway and Killultagh
+George Keith,Franciscus Mercurius van Helmont
+George Keith,William Penn
+```
+
+Maintenant que vous avez téléchargé les données Quaker et vu comment elles sont structurées, il est temps de commencer à travailler avec ces données à l'aide de Python. Une fois Python et pip installés (voir [Prérequis](#prérequis) ci-dessus), vous devez installer NetworkX, en tapant ceci dans votre ligne de commande (voir notre leçon disponible [en anglais](/en/lessons/intro-to-bash) et [en espagnol](/es/lecciones/introduccion-a-bash))[^4]&nbsp;:
```

Comment:
At the end of ligne 108, we need to remove the link to the EN and ES lessons and add an FR link, as the FR lesson about Bash was just published :), like so:

(voir [notre leçon sur les interfaces en ligne de commande](/fr/lecons/intro-a-bash-et-zsh)[^4]&nbsp;: